engine/jobmaster/dm/api.go

```diff
@@ -102,6 +115,7 @@ func (jm *JobMaster) QueryJobStatus(ctx context.Context, tasks []string) (*JobSt
 				ExpectedStage: expectedStage,
 				WorkerID:      workerID,
 				Status:        queryStatusResp,
+				CfgModRevison: cfgModRevision,
```

Comment:
Now we added ExpectedCfgModRevison and CfgModRevison, the OpenAPI client needs to parse and check their equivalent if it should ignore some task status. Can we add a "outdated stage" or change the queryStatusResp to indicate it's outdated?

Reply:
I think version is more intuitive than outdated.🤔

Reply:
the client is more complex to handle it. I'll write integration test soon, and for each response I must check the outer revision is same as inner revision. This is not easy as just read one line "queryStatusResp"

But on the other hands, user may need full information to his/her own client logic.

Reply:
For integration test, I think we can just check the number of version, because it's increase one per update request now.

Reply:
My use case is when we need to check if there are two "Running" tasks, we still need to parse and compare revisions, otherwise we don't have confident the "Running" is not outdated. In fact I think all response check should contain this logic, very complicated

Reply:
Is this ok?
```
{
    expect-mod-revison: 4
    source1: {
        cfgModRevision: 3(outdated)
    }
}
```

Reply:
can add another field, so (xxx-)cfg-mod-revision is all number.

Now I think the complexity is that task status can't be determinated within the scope of itself. It's OK for me when we don't need to check job status to determine task status.

Reply:
remove cfg-mod-revison and add config-outdated